Sources/NautilusTelemetry/Tracing/Span+URLSession.swift

```diff
@@ -21,6 +21,24 @@ public extension Span {
 		return request.httpMethod ?? "HTTP"
 	}
 
+	/// Add `traceparent` header to a URLRequest if we're sampling
+	/// - Parameter isSampling: whether we are sampling, defaults to InstrumentationSystem.tracer.isSampling
+	/// - Parameter urlRequest: urlRequest to modify
+	func addTraceHeadersIfSampling(_ urlRequest: inout URLRequest, isSampling: Bool = InstrumentationSystem.tracer.isSampling) {
+		if isSampling {
+			let value = traceParentValue(sampled: true)
+			urlRequest.addValue(value.1, forHTTPHeaderField: value.0)
+		}
+	}
+
+	/// Add `traceparent` header to a URLRequest regardless of sampling state
+	/// Sampled flag determined from InstrumentationSystem.tracer.isSampling
```

Comment:
Like `addTraceHeadersIfSampling`, `InstrumentationSystem.tracer.isSampling` is used for the default, but it might not determine the sampled flag.